### DISCUSSION TITLE
How can I have a vless vision reality balancer?

### DISCUSSION THREAD

**Discussion by MrVb0** [XTLS/Xray-core#2156] [Q&A]
How can I have a vless vision reality balancer to connect one configuration to multiple reality servers?
Please help me in writing its config

**Comment by us254** [comment]
https://github.com/XTLS/Xray-core/issues/2097#issuecomment-1556400225

**Reply by MrVb0** [reply]
thanks
https://github.com/XTLS/Xray-core/discussions/2194

**Comment by GreatMichaelLee** [comment]
balancer还是RR的把，能否实现一个简单的weight调度呢。
